spec/site/tlc/modes_spec.rb

```diff
@@ -179,8 +179,7 @@
     Validator::Site.connected do |task,supervisor,site|
       prepare task, site
       switch_yellow_flash 1
-      task.sleep(90)
-      wait_normal_control
+      wait_normal_control 120
```

Comment:
the yellow flash timeout used in 1 minute, ie. 60 seconds. i guess a little bit of buffer is reasonable, but do we need 30 seconds? i think this extra time should be a test config, not hardcoded